app/views/exams/components/ExamLevel.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <span v-if="index">{{ index }}.</span>
+    <a
+      :href="levelLink"
+      target="_blank"
+    >{{ makeTitle(level.slug) }}</a>
```

Comment:
actually they just requested that we don't show the level names in the exam 😅  - can we instead just show `Level ${index}`